src/domain/proof-of-liabilities/generate-liabilities-tree.ts

```diff
@@ -0,0 +1,33 @@
+import { getHash } from "@domain/proof-of-liabilities/generate-leaves"
+import { toSats } from "@domain/bitcoin"
+
+const getSum = (
+  val1: CurrencyBaseAmount,
```

Comment:
I think we can also do proof of reserve fot bitcoin, not really for USD equivalent